core/client/tests/integration/services/ajax-test.js

```diff
@@ -5,6 +5,7 @@ import {
 } from 'ember-mocha';
 import Pretender from 'pretender';
 import wait from 'ember-test-helpers/wait';
```

Comment:
Good catch, I'll take that out. Wonder why my editor linter didn't highlight that 🤔